src/NuGet.Core/NuGet.Commands/PushRunner.cs

```diff
@@ -50,6 +50,7 @@ public static class PushRunner
                 timeoutSeconds,
                 disableBuffering,
                 endpoint => CommandRunnerUtility.GetApiKey(settings, endpoint, apiKey),
+                symbolsEndpoint => CommandRunnerUtility.GetApiKey(settings, symbolsEndpoint, symbolsApiKey),
```

Comment:
Actually even before my change, the API key lookup would use the symbols source, not nuget.org.
PackageUpdateResource.Push would call getApiKey passing in symbolsSource.

What's different now is this command: "nuget.exe push foo.nupkg -source nuget.org -apikey XXX"
Before, XXX would be used for the symbols apikey. After my current change, it won't. I can fix that.

---
In reply to: [68460573](https://github.com/NuGet/NuGet.Client/pull/712#discussion_r68460573) [](ancestors = 68460573,68441601)